js/src/personal/kelly/Kelly.module.css

```diff
@@ -1,26 +1,9 @@
-.body_container{
-  display: flex;
-  align-items: center;
-  justify-content: center;
-  height: 50vw;
-  background-color: #fdf7ee;
-  background-image: url("https://i.pinimg.com/564x/8a/28/bd/8a28bd8947deb747aba8b2557fb41982.jpg");
-  background-position: center;
-  background-repeat: no-repeat;
+html{
```

Comment:
We never figured out how this works without setting it globally right?

You might be able to use what Mali used with Mantine's scrollIntoView()
https://mantine.dev/hooks/use-scroll-into-view/